alkemio_virtual_contributor_engine/alkemio_vc_engine.py

```
@@ -0,0 +1,48 @@
+        self.rabbitmq = RabbitMQ()
+        self.handler = None
+
+    async def start(self):
+        await self.rabbitmq.connect()
+        if self.invoke_handler is not None:
+            await self.rabbitmq.consume(self.invoke_handler)
+        await asyncio.Future()
+
+    async def invoke_handler(self, message: AbstractIncomingMessage):
+        logger.info("New message recieved.")
+        if not self.handler is None:
```

Ruff

Test for object identity should be `is not`

Convert to `is not`

(E714)